### PR TITLE
move from AppVeyor to Github Action

### PR DESCRIPTION
**Pull request by mrpalide**
Fixes #_	

 Changes:	
- move from AppVeyor to Github Action

How to test this PR:
_

### CODE REVIEW

File: pkg/noise/read_writer.go [hunk 1]
```diff
@@ -20,7 +21,7 @@ const MaxWriteSize = maxPayloadSize
 
 // Frame format: [ len (2 bytes) | auth & nonce (24 bytes) | payload (<= maxPayloadSize bytes) ]
 const (
-	maxFrameSize   = 4096                                 // maximum frame size (4096)
+	maxFrameSize   = 65536                                // maximum frame size (4096)
```
Comment (ersonp) [inline pkg/noise/read_writer.go]: I think we should do this in a different PR unless you have tested it with skywire and everything is working correctly. If you have then just change the comment to `// maximum frame size (65536)`

Comment (mrpalide) [reply pkg/noise/read_writer.go]: Yeah, you're right. Revert it to 4096. Will make other PR for this.

**Review by ersonp — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,4 +12,5 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - add `gen` command to generate config, with two flag `-o` for output file and `-t` for using test env values
 
 ### Changed
-- TBD
+- increase limitation of ReadWritePacket on noise pkg from 4KB to 64KB
```
Comment (ersonp) [inline CHANGELOG.md]: This is n o longer needed
